README.md

```
@@ -34,6 +34,7 @@ ______________________________________________________________________
   - [Class deprecation](#class-deprecation)
   - [Deprecating constants and instances](#deprecating-constants-and-instances)
   - [Deprecating Enums and dataclasses](#deprecating-enums-and-dataclasses)
+  - [Typing deprecated proxies](#typing-deprecated-proxies)
```

markdownlint-cli2 (0.23.2)

[warning] Link fragments should be valid

(MD051, link-fragments)